src/%IPM/Main.cls

```diff
@@ -1910,7 +1910,7 @@ ClassMethod GetListModule(ns = {$namespace}, ByRef Mod, OnlyName = 1) As %Status
 	set in=""
 	While tRes.%Next(.tSC) {
 		$$$ThrowOnError(tSC)
-		Set name=tRes.%Get("Name")
+		Set name=$Select(tRes.%Get("DisplayName")="": tRes.%Get("Name"), 1: tRes.%Get("DisplayName"))
```

Comment:
Instead of doing this everywhere, could simplify a bit and have 2 new properties storage class:
DisplayNameInternal and DisplayName. DisplayNameInternal will be the one that gets set every time Name is set.
DisplayName will be a calculated property with the following SqlComputeCode:
`SqlComputeCode = { Set {*} = $Case({DisplayNameInternal}, "": {Name}, : {DisplayNameInternal}  }`

Reply:
Good point! Also moved the NameSet method one level upper (from %IPM.Storage.Module to %IPM.General.ModuleInfo)